### PR TITLE
feat(variable-labeling): Update the documentation so that conflicting…

### PR DESCRIPTION
**Pull request by andromaqui**
… variable labeling behaviour is underlined

relates to: OPT-5966

### CODE REVIEW

File: docs/components/optimize/userguide/additional-features/variable-labelling.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+id: variable-labeling
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labelling.md:2]: There looks like a typo in the filename ("labelling")

File: docs/components/optimize/userguide/additional-features/variable-labelling.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: Variable Labeling
+description: Add a label for a variable in a process definition.
+---
+
+The variable labeling functionality allows users to add, update and delete batches of variable labels. This allows Optimize to display a variable's label instead of its original name, anywhere the given process definition is being used. Some examples of that would be
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labelling.md:7]: The UI says "rename" and "alias", but not "label". We should try to consolidate terms whenever possible and avoid introducing new ones.
I'm comfortable replacing "alias" with "label" if it improves usability (@asiamalina / @christinaausley )

File: docs/components/optimize/userguide/additional-features/variable-labelling.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+in reports, configuring filters, report grouping, dashboard filters and Event Based Processes.
+
+In order to use this feature, go on the definition edit window from inside a report and click on **Edit Variables** to access the label edit panel. You will then see the following panel:
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labelling.md:10]: Syntax - should be "go to the" definition edit window

File: docs/components/optimize/userguide/additional-features/variable-labelling.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+![Label Edit panel](./img/variable-labeling-panel.png)
+
+A label can be deleted by simply inputting an empty field for its value.
+
+## Limitations
+
+Please note that this feature is currently not supported in outlier analysis and csv export. This means that during Outlier Analysis, the original name of a variable will be displayed.
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labelling.md:18]: We should use consistent case when referring to Outlier Analysis

File: docs/components/optimize/userguide/additional-features/variable-labelling.md (hunk 2 above)
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labelling.md:7]: recommend adding the use case
...batches of variable labels **so your data is more understandable by business users**

**Review by HanselIdes — DISMISSED** [review]
This looks good. I raised a few comments with basic info.

If we wanted to improve it further, we could continue tweaking the wording for clarity and to be more succinct. I'd recommend @christinaausley / @akeller make that decision on how much time they want to spend

**Comment by christinaausley** [comment]
I made a few adjustments in my latest commit, some of which @HanselIdes also commented on. My commit included the following:

- Ensuring anything that is not directly a product is lower case.
- Adding a note admonition.
- Utilizing Oxford comma.

Let me know if you have any questions -- nice work!

File: docs/components/optimize/userguide/additional-features/variable-labelling.md (hunk 2 above)
Comment (andromaqui) [reply docs/components/optimize/userguide/additional-features/variable-labelling.md]: I think Omran has a ticket to update this to "label" everywhere in the UI

File: docs/components/optimize/userguide/additional-features/variable-labeling.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+id: variable-labeling
+title: Variable labeling
+description: Add a label for a variable in a process definition.
+---
+
+The variable labeling functionality allows users to add, update, and delete batches of variable labels so your data is more understandable by business users. This allows Optimize to display a variable's label instead of its original name anywhere the given process definition is being used. Some examples of that would be
+in reports, configuring filters, report grouping, dashboard filters, and event-based processes.
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/additional-features/variable-labeling.md:8]: Recommend: "when viewing and configuring reports, dashboards, or event-based processes"

**Review by HanselIdes — APPROVED** [review]
Approved, assuming you make the last text change. No further review is required from my side

File: docs/components/optimize/userguide/additional-features/variable-labeling.md (hunk 5 above)
Comment (andromaqui) [reply docs/components/optimize/userguide/additional-features/variable-labeling.md]: updated !

**Review by HanselIdes — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**